common/v2/config/routeUrls.ts

```diff
@@ -0,0 +1,128 @@
+import { IRoutePath, IRoutePaths } from 'v2/types';
+
+const PATHS: IRoutePath[] = [
+  {
+    name: 'ROOT',
+    path: '/'
+  },
+  {
+    name: 'BUY',
+    title: 'Buy Crypto',
+    path: 'https://buy.mycrypto.com/'
+  },
+  {
+    name: 'PURCHASE_TREZOR',
+    path: 'https://shop.trezor.io/?offer_id=10&aff_id=1735'
```

Comment:
Oh dear I didn't see the whole `data.tsx` and `links.tsx` combo.

Reply:
Ok, to avoid refactoring too far I duplicated some code to handle external links, I removed them from this file and created an EXT_URLS object that follows the same pattern. For the moment I won't fix the other external links.